src/actions/ProvidersActions.ts

```diff
@@ -0,0 +1,45 @@
+import { Notify } from 'quasar';
+import { getListOfProvidersQuery } from '../services';
+import {
+  useQuery,
+  actionCallbackReturnTypes,
+  actionCallbackParamsTypes,
+} from '../utils/apollo';
+
+export const getListOfProviders = ({
+  limit = 10,
+  offset = 0,
+  query = '',
+}: actionCallbackParamsTypes) => {
+  const name = `%${query}%`;
```

Comment:
In this case, the name const can be just above variables, as is not used anywhere

Reply:
Good observation